.bundle/config

```diff
@@ -1,2 +0,0 @@
----
-BUNDLE_PATH: "vendor/bundle"
```

Comment:
This alters the way in which gems are installed (Unless it's set elsewhere). Are we happy to do this?

EDIT: Seen we've altered the dockerfile. But this will affect people running locally. Can we keep this, and the dockerfile changes, or would that be the issue you were describing?

Reply:
From the running process in the container's point of view, it doesn't mean anything. So long as the path matches between install and exec.

Heroku's recommendation is not to check in the .bundle/config file (https://devcenter.heroku.com/articles/bundler-configuration#:~:text=bundle%2Fconfig%20file.,to%20configure%20Bundler%20on%20Heroku.) and I agree with them.

It could go in the .dockerignore and .gitignore files so people can still use a local copy of the file if they want, instead of user-config or environment variables?

Reply:
I think that sounds a really good idea. In gitignore. Then it can be just like `.env` I'm happy with that approach.